pydoctor/templatewriter/pages/functionchild.py

```diff
@@ -57,7 +57,10 @@ def decorator(self, request, tag):
     @renderer
     def functionDef(self, request, tag):
         def_stmt = 'async def' if self.ob.is_async else 'def'
-        return [def_stmt, ' ', self.ob.name, signature(self.ob), ':']
+        name = self.ob.name
+        if name.endswith('.setter') or name.endswith('.deleter'):
+            name = name[:name.rindex('.')]
```

Comment:
Minor comment:
At least for the setter, if you see that a property has no setter you can think that it's read-only.

But maybe, if a deleter is defined and has no docstring, it can't be skipped from the docs?

Reply:
It might still be useful information to know that the property can be deleted. Also the source link for the deleter would be missing then, although in practice getter/setter/deleter often get defined close to each other, so that's not a big problem.

I don't think deleters are common: there are zero in Twisted and I can't remember ever having used one myself. I had even forgotten they existed until I re-read the language documentation when working on this PR.

I think that in the future we probably want to revisit how properties are presented and create some kind of unified display that combines information from the getter, setter and deleter in a single output box. But that's too ambitious to do right now.